### ISSUE TITLE
error loading pre-trained weights for fine-tuning YOLONAS

### ISSUE THREAD

**Issue by albertofernandezvillan** [Deci-AI/super-gradients#949]
Instantiating a YOLONAS model for fine-tuning:

`model = models.get('yolo_nas_l', num_classes=len(dataset_params['classes']), pretrained_weights="coco")`

Line above works, but following line (just copy/paste the download weights to the current working directory):

`model = models.get('yolo_nas_l', num_classes=len(dataset_params['classes']), pretrained_weights="coco", checkpoint_path="./yolo_nas_l_coco.pth")`

Throws an error loading the weights of the head (as my dataset has different number of classes).
I want to load the weights from a file on disk for fine-tuning.

How to cope with this error?

**Comment by Louis-Dupont** [comment]
Hi @albertofernandezvillan 

I'm gonna try to reproduce this very soon, but id like to make sure, is "./yolo_nas_l_coco.pth" the path to the checkpoints you just got, or is it after fine-tuning?
